### DISCUSSION TITLE
Which option is equivalent to ST_Area(geography geog, boolean use_spheroid=true)?

### DISCUSSION THREAD

**Discussion by tiagobaranzelli** [NetTopologySuite/NetTopologySuite#767] [Q&A]
I wonder if there is an option equivalent to `ST_Area(geography geog, boolean use_spheroid=true)` as mentioned on this page https://postgis.net/docs/ST_Area.html

**Comment by FObermaier** [comment]
There is none. NTS' computes area on planar 2D coordinate system.
